.github/workflows/ci-test-limited.yml

```
@@ -48,7 +48,13 @@ jobs:
         ports:
           # Opens tcp port 6379 on the host and service container
           - 6379:6379
+      postgres:
+        if: contains(github.base_ref, 'pg') || github.ref_name == 'pg'
+        image: postgres:14
+        ports:
+          - 5432:5432
       mongo:
+        if: "!contains(github.base_ref, 'pg') && github.ref_name != 'pg'"
```

actionlint

unexpected key "if" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)